zebra-state/src/service.rs

```diff
@@ -1255,10 +1253,7 @@ impl Service<ReadRequest> for ReadStateService {
                         Ok(ReadResponse::BestChainNextMedianTimePast(median_time_past?))
                     })
                 })
-                .map(|join_result| {
-                    join_result.expect("panic in ReadRequest::BestChainNextMedianTimePast")
-                })
-                .boxed()
+                .wait_for_panics()
```

Comment:
Optional/Nitpick: It may be nice to pass in arguments for log or panic messages, but `#[track_caller]` should be enough.